### PR TITLE
added color consistency for svg icons

### LINKED ISSUE #407: Add Icon Color Consistency
Currently some of the icons are the same colors and others are just the colors natural to their normal context.
I suggest making them all use the same color scheme, the icon colors are not especially useful for scanning the categories IMO and I believe it looks cleaner when consistent.

Before:
Screen Shot 2020-07-30 at 11 51 12 AM

After:
Screen Shot 2020-07-30 at 11 51 56 AM

### PR DESCRIPTION
**Pull request by Neilblaze**
**Description**
Added color consistency for svg icons. Note that, for the `taskcluster` svg, I was unable to fill the black bg into white. So, I had to keep the black bg color for that svg. ~~Also, as backup, I kept the default/original `taskcluster` svg in `./src/images/projectIcons/default_img`. Rest, everything is fixed :)~~

Fixes #407 

Signed-off-by: Pratyay Banerjee <48355572+Neilblaze@users.noreply.github.com>

### CODE REVIEW

**Comment by Neilblaze** [comment]
oh sorry, I forgot to include the `/default_img` folder in node modules

File: src/images/projectIcons/rust.svg [hunk 1]
```diff
@@ -1 +1,19 @@
...
...4.444 -10.941,0 c -0.949,-3.336 -1.458,-6.857 -1.458,-10.496 0,-1.389 0.075,-2.762 0.219,-4.111 z m 29.38,-2.374 0,-7.205 12.914,0 c 0.667,0 4.71,0.771 4.71,3.794 0,2.51 -3.101,3.41 -5.651,3.41 l -11.973,0 z m 46.932001,6.485 c 0,0.956 -0.035,1.902 -0.105,2.841 l -3.926,0 c -0.393,0 -0.551,0.258 -0.551,0.643 l 0,1.803 c 0,4.244 -2.393,5.167 -4.49,5.402 -1.997001,0.225 -4.211001,-0.836 -4.484001,-2.058 -1.178,-6.626 -3.141,-8.041 -6.241,-10.486 3.847,-2.443 7.85,-6.047 7.85,-10.871 0,-5.209 -3.571,-8.49 -6.005,-10.099 -3.415,-2.251 -7.196,-2.702 -8.216,-2.702 l -40.603,0 c 5.506,-6.145 12.968,-10.498 21.408,-12.082 l 4.786,5.021 c 1.082,1.133 2.874,1.175 4.006,0.092 l 5.355,-5.122 c 11.221,2.089 20.721,9.074 26.196001,18.657 l -3.666,8.28 c -0.633,1.433 0.013,3.109 1.442,3.744 l 7.058,3.135 c 0.122,1.253 0.186,2.519 0.186,3.802 z m -40.575001,-41.88 c 1.24,-1.188 3.207,-1.141 4.394,0.101 1.185,1.245 1.14,3.214 -0.103,4.401 -1.24,1.188 -3.207,1.142 -4.394,-0.102 -1.184,-1.243 -1.139,-3.212 0.103,-4.4 z m 36.380001,29.277 c 0.695,-1.571 2.531,-2.28 4.1,-1.583 1.569,0.696 2.277,2.536 1.581,4.107 -0.695,1.572 -2.531,2.281 -4.101,1.584 -1.567,-0.696 -2.275,-2.535 -1.58,-4.108 z" /></svg>
```
Comment (helfi92) [inline src/images/projectIcons/rust.svg:19]: Any idea why this went from a single tag to using both an xml and svg tags?

File: src/images/projectIcons/pontoon.svg [hunk 2]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="512" height="512" viewBox="0 0 512 512" shape-rendering="geometricPrecision"><path fill="#fff" d="m32 0h480v512h-480z"/><g fill="none"><path stroke="#272a2f" stroke-width="192" d="m192 512v-352h160v192h-160"/><path stroke="#7bc876" stroke-width="64" d="m192 512v-352h160v192h-160"/></g></svg>
+<svg xmlns="http://www.w3.org/2000/svg" width="512" height="512" viewBox="0 0 512 512" shape-rendering="geometricPrecision"><path fill="#fff" d="m32 0h480v512h-480z"/><g fill="#4bacb8"><path stroke="#4bacb8" stroke-width="192" d="m192 512v-352h160v192h-160"/><path stroke="#fff" stroke-width="64" d="m192 512v-352h160v192h-160"/></g></svg>
```
Comment (helfi92) [inline src/images/projectIcons/pontoon.svg:1]: Looks nice!

File: src/images/projectIcons/seamonkey.svg [hunk 3]
```diff
@@ -1 +1 @@
...
...59 76.75c1.9 2.28 3.5 4.8 4.88 7.43-1.64 1.32-3.47 2.37-5.3 3.39-.46-3.33-.65-6.71-.45-10.07l.87-.75z"/><path fill="#4bacb8" d="M77.15 81.67c1.6 2.2 2.43 4.82 3.28 7.37-1.29.64-2.58 1.26-3.89 1.87-.48-3.08-.36-6.25.61-9.24z"/><path fill="#4bacb8" d="M66.65 83.31c2.03-.5 3.47.02 4.32 1.57-1.59 1.1-3.26 3.04-5.44 2.24.36-1.28.73-2.55 1.12-3.81z"/><path fill="#4bacb8" d="M69.09 93.78c.18-3.03.8-6.12 2.81-8.5 1.01 2.18 1.42 4.54 1.71 6.91-1.49.57-3 1.1-4.52 1.59z"/><path fill="#4bacb8" d="M61.36 86.96c1.54-1.5 3.49 1.06 1.3 1.83-1.57 1.48-3.29-.88-1.3-1.83z"/><path fill="#4bacb8" d="M56.21 91.5c3.65-.62 7.13-1.86 10.49-3.37.36 2.03.63 4.07.84 6.13-8.2 2.23-16.5 4.14-24.92 5.29-.37-.19-1.12-.57-1.49-.77 2.82-1.42 5.03-4.41 8.42-4.34 1.05-.99 2.07-2.01 3.08-3.04 1.19.07 2.38.19 3.58.1z"/><path fill="#4bacb8" d="M105.28 100.24c3.99-2.12 7.57-5.9 12.41-5.61-8.53 14.17-22.49 25.47-38.79 29.17-6.32 1.61-12.88 1.5-19.34 1.35 7.32-2.34 13.71-6.68 20.43-10.28 8.43-4.88 16.78-9.9 25.29-14.63z"/><path fill="#4bacb8" d="M40.35 102.49c10.5-1.22 20.97-3.12 30.99-6.58-1.14 2.71-4.16 3.06-6.54 4.11-10.62 5.13-19.97 12.65-30.98 17.03-2.37-1.42-4.59-3.07-6.82-4.7 4.52-3.19 8.81-6.72 13.35-9.86z"/></svg>
```
Comment (helfi92) [inline src/images/projectIcons/seamonkey.svg:1]: Screen Shot 2020-09-30 at 9 23 14 AM

Screen Shot 2020-09-30 at 9 19 14 AM

I'm not sure how I feel about this transformation. For complex logos, perhaps it's best to leave them as is.

File: src/images/projectIcons/servo.svg [hunk 4]
```diff
@@ -1 +1 @@
...-.137.416-.242.606.12-.478.178-.993.09-1.485-.072-.404-.186-.94-.015-1.328-.19.357-.279.769-.38 1.105 0-.694-.003-1.293.383-1.89.174-.27.34-.612.537-.852-.005.227.034.542.071.809.049.352-.02.701-.003 1.05.122-.431.44-.774.55-1.22.09-.36.08-.725.14-1.083.138.269.263.699.23 1.006.304-.85.395-1.595.26-2.485a8.78 8.78 0 0 1-.094-1.321c0-.206-.028-.472.09-.64 0 .43.428.802.475 1.198.028-.604.441-1.165.452-1.783-.237-.154-.448-.466-.712-.486-.37-.027-.323.276-.579.359-.264.085-.468-.024-.756-.06-.408-.053-.908.009-1.327.065a.61.61 0 0 0-.059.012c-.179.043-.383.161-.552.155-.312-.012-.529-.273-.8-.373-.412.64-.872.157-1.227-.213-.238.143-.316.609-.572.724-.434.194-.434-.178-.743-.36-.395-.233-.114-.145-.533.11-.272.164-.686.492-.812-.003a.516.516 0 0 1-.081.063l-.014.009-.047.03h-.002l-.053.029-.017.008a2.082 2.082 0 0 1-.042.02l-.018.01a2.414 2.414 0 0 1-.191.08l-.018.007a6.456 6.456 0 0 1-.133.049l-.017.006a11.73 11.73 0 0 1-.053.019l-.014.005a9.943 9.943 0 0 1-.05.017c-.097.033-.188.064-.26.09l-.022.008c-.075.025-.154.04-.234.05-.273.031-.565-.004-.792.091-.28.118-.356.53-.694.63-.135.04-.333-.127-.473-.054-.403.207-.321.8-.69 1.073-.45-.267-.277-.552-.897-.33-.26.094-.638.266-.867.422
```
Comment (helfi92) [inline src/images/projectIcons/servo.svg:1]: Screen Shot 2020-09-30 at 9 19 03 AM

Screen Shot 2020-09-30 at 9 19 07 AM

Same here. Probably best to just use the original.

File: src/images/projectIcons/taskcluster.svg [hunk 5]
```diff
@@ -1 +1 @@
...
...troke-linecap="square"/><path style="marker:none" d="M34.465 14.304a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M29.868 16.502l-3.343 4.144" fill="none" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square"/><path style="marker:none" d="M15.964 30.906a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM17.65 38.585a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM25.487 35.637a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM43.251 29.417a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M20.562 26.046l-5.06 3.313M21.825 27.277l-5.512 8.906M23.521 27.862l-.548 4.968M27.824 25.127l9.929 3.38" fill="none" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square"/><path style="marker:none" d="M32.735 39.336a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0zM35.177 31.659a2.824 2.824 0 1 0-5.648 0 2.824 2.824 0 0 0 5.648 0z" color="#4bacb8" fill="#4bacb8" overflow="visible"/><path d="M26.942 26.596l3.349 3.133M25.425 27.618l3.476 9.08" stroke="#4bacb8" stroke-width="1.1297048" stroke-linecap="square" fill="none"/></g></svg>
```
Comment (helfi92) [inline src/images/projectIcons/taskcluster.svg:1]: Taskcluster actually has a new logo https://github.com/taskcluster/taskcluster-media/blob/main/source/logo/logo.svg

You can try using the new one and see if it looks better.

File: src/images/projectIcons/webextensions.svg [hunk 6]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 64 64"><defs><linearGradient id="a" x1="0%" y1="0%" x2="0%" y2="100%"><stop offset="0%" stop-color="#4bacb8"/><stop offset="100%" stop-color="#4bacb8"/></linearGradient></defs><path fill="url('#a')" d="M42 62c2.2 0 4-1.8 4-4V43.8s.4-3.7 2.8-3.7S51 44 55.5 44c2.3 0 6.2-1.2 6.2-8.2 0-7-3.9-7.9-6.2-7.9-4.5 0-4.3 3.7-6.7 3.7-2.4 0-2.8-3.8-2.8-3.8V22c0-2.2-1.8-4-4-4H31.5s-3.4-.6-3.4-3 3.8-2.6 3.8-7.1c0-2.3-1.3-5.9-8.3-5.9s-8 3.6-8 5.9c0 4.5 3.4 4.7 3.4 7.1 0 2.4-3.4 3-3.4 3H6c-2.2 0-4 1.8-4 4v7.8s-.4 6 4.4 6c3.1 0 3.2-4.1 7.3-4.1 2 0 4 1.9 4 6 0 4.2-2 6.3-4 6.3-4 0-4.2-4.1-7.3-4.1-4.8 0-4.4 5.8-4.4 5.8V58c0 2.2 1.8 4 4 4h13s6.3.4 6.3-4.4c0-3.1-4-3.6-4-7.7 0-2 2.2-4.5 6.4-4.5s6.6 2.5 6.6 4.5c0 4-3.9 4.6-3.9 7.7 0 4.9 6.3 4.4 6.3 4.4H42z"/></svg>
```
Comment (helfi92) [inline src/images/projectIcons/webextensions.svg:1]: Looks nice!

**Review by helfi92 — CHANGES_REQUESTED** [review]
Nice work :) Added some minor requested changes.

File: src/images/projectIcons/rust.svg (hunk 1 above)
Comment (Neilblaze) [reply src/images/projectIcons/rust.svg:19]: I think that was done for the ease of differentiation.

File: src/images/projectIcons/pontoon.svg (hunk 2 above)
Comment (Neilblaze) [reply src/images/projectIcons/pontoon.svg:1]: Thanks :smile:

File: src/images/projectIcons/seamonkey.svg (hunk 3 above)
Comment (Neilblaze) [reply src/images/projectIcons/seamonkey.svg:1]: Ok I'll leave that as it is!

File: src/images/projectIcons/servo.svg (hunk 4 above)
Comment (Neilblaze) [reply src/images/projectIcons/servo.svg:1]: Ok!

File: src/images/projectIcons/taskcluster.svg (hunk 5 above)
Comment (Neilblaze) [reply src/images/projectIcons/taskcluster.svg:1]: Ok, I'll try it for sure :smile:

File: src/images/projectIcons/webextensions.svg (hunk 6 above)
Comment (Neilblaze) [reply src/images/projectIcons/webextensions.svg:1]: Thanks :smiley:

**Comment by Neilblaze** [comment]
But like I don't know why the Netlify deployment is failing. Maybe it's an issue of Netlify. Nonetheless, I'll make the requested changes by tomorrow :rocket:

**Pull request merged**